app/javascript/mastodon/features/list_editor/components/edit_list_form.js

```diff
@@ -57,6 +65,11 @@ class ListForm extends React.PureComponent {
           onChange={this.handleChange}
         />
 
+        <label htmlFor='is-exclusive-checkbox'>
+          <Toggle className='is-exclusive-checkbox' defaultChecked={isExclusive} onChange={this.handleToggle} />
+          <FormattedMessage id='lists.is-exclusive' defaultMessage='Exclusive?' />
```

Comment:
```suggestion
          <FormattedMessage id='lists.is_exclusive' defaultMessage='Exclusive?' />
```
For consistency the id should use an underscore. Or just be "lists.exclusive".

I also propose the defaultMessage to be "Hide posts from these users in home" (or similar).
This requires changes of styling.
See https://github.com/glitch-soc/mastodon/pull/1996#issuecomment-1356164327 for how it could look.